clients/egui/src/account/modals/mod.rs

```diff
@@ -66,6 +66,10 @@ impl super::AccountScreen {
                 use SettingsResponse::*;
                 match inner {
                     SuccessfullyUpgraded => self.refresh_sync_status(ctx),
+                    ToggleToolbarVisibility(_) => {
+                        self.modals.settings = None;
```

Comment:
avoids a node none existent bug thrown by accesskit which forbids hiding elements if they're disabled. I enable the toolbar by disabling the settings modal and then I reload the tab. this has the negative effect of closing the settings prematurely.